sdk/core/System.Net.ClientModel/tests/clients/System.Net.ClientModel.Tests.Clients.csproj

```diff
@@ -1,19 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
+    <Description>Test client code for use in System.Net.ClientModel test libraries.</Description>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
```

Comment:
Should set test support project to false.  https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L20  It is auto added as true if you are in the test folder and don't end with `.Tests`

Same with internal test project below https://github.com/Azure/azure-sdk-for-net/pull/39368/files#diff-79755eaf51f7bed3fd9e6ffa2c25c504ea434aa8eb54be83b24fa2f3b463447dR6

Reply:
What does that flag mean/do?

Reply:
It sounds like if we add this flag, I don't need to add the entry to projects.txt, so I'm cool with that, and will learn more about it on my own time 🙂

Reply:
Actually, I think we want it to be true to match Azure.Core.Tests.Common -- I'll add that explicitly, and we can revisit if we want to start adding tests here that run in CI's.